node/packages/parser/src/ethereum/uniV2.ts

```diff
@@ -117,4 +60,76 @@ export class Parser {
     const initCodeHash = '0x96e8ac4277198ff8b6f785478aa9a39f403cb768dd02cbee326c3e7da348845f' // https://github.com/Uniswap/v2-periphery/blob/dda62473e2da448bc9cb8f4514dadda4aeede5f4/contracts/libraries/UniswapV2Library.sol#L24
     return ethers.utils.getCreate2Address(factoryContract, salt, initCodeHash)
   }
+
+  private async getTransfers(tx: Tx): Promise<Transfer[] | undefined> {
+    const data = tx.ethereumSpecific?.data
+    if (!data) return
+    const sendAddress = tx.vin[0].addresses?.[0] ?? ''
+    return await (async () => {
```

Comment:
No reason to wrap the switch in another anon async func.

Reply:
Addressed in https://github.com/shapeshift/unchained/pull/267/commits/5fecfd3a2687cfdb37e4b0d571c38d4bcca7b628